src/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "types": ["node"],
     "esModuleInterop": true,
     "declaration": true,
+    "ignoreDeprecations": "5.0",
```

Comment:
This is a quick way to ignore error messages in the console in development, but we want to make sure we catch things that shouldn't be slipping through so we can update dependencies and the code when necessary with relative ease.

Reply:
This still needs to be removed.